Audacia.Log.AspNetCore/CHANGELOG.md

```diff
@@ -1,8 +1,27 @@
 ﻿# Changelog
 
+### v4.0.0 - 2024-10-07
+
+### Added
```

Comment:
Wonder if there is a "Breaking" heading or something, i.e. the one you renamed could be listed under breaking changes

Reply:
I think the v4 does imply there is a breaking change yeah - I'm meaning this doesn't say what the breaking changes is.

I'm suggesting we have the following:

```
### v4.0.0 - 2024-10-07

### Breaking

- Renamed `AddRequestBodyTelemetry` to `AddActionRequestBodyTelemetry` and setup from `service.AddRequestBodyTelemetry()` to `services.AddActionRequestBodyTelemetry` 

### Added

- A new telemetry initialiser and Action filter for enrich response body from ASPNET API actions. (
  See `LogResponseBodyActionTelemetryInitialiser` and `LogResponseBodyActionFilterAttribute`)
- A new telemetry initialiser to enrich dependency telemetry with the request and response body if configured. (
  See `HttpDependencyBodyCaptureTelemetryInitializer`)

### Changed

- Added configuration to enforce code analysis for `Audacia.Log`, `Audacia.Log.AspNetCore`
  and `Audacia.Log.AspNetCore.Tests`.
    - Fixed all violation from enforce static analysis.
```

Reply:
Okay - will update

Reply:
Spoke to Alex about this, I think the breaking changes should be proceeded by `**Breaking:**`

So I think it would be this:

```
### v4.0.0 - 2024-10-07

### Added

- A new telemetry initialiser and Action filter for enrich response body from ASPNET API actions. (
  See `LogResponseBodyActionTelemetryInitialiser` and `LogResponseBodyActionFilterAttribute`)
- A new telemetry initialiser to enrich dependency telemetry with the request and response body if configured. (
  See `HttpDependencyBodyCaptureTelemetryInitializer`)

### Changed

- **Breaking:** Renamed `AddRequestBodyTelemetry` to `AddActionRequestBodyTelemetry` and setup from `service.AddRequestBodyTelemetry()` to `services.AddActionRequestBodyTelemetry` 
- Added configuration to enforce code analysis for `Audacia.Log`, `Audacia.Log.AspNetCore`
  and `Audacia.Log.AspNetCore.Tests`.
    - Fixed all violation from enforce static analysis.
```